chamber/models/__init__.py

```diff
@@ -376,12 +391,17 @@ def _post_save(self, *args, **kwargs):
     def _call_post_save(self, *args, **kwargs):
         self._post_save(*args, **kwargs)
 
-    def save(self, *args, **kwargs):
+    def save_simple(self, *args, **kwargs):
+        super(SmartModel, self).save(*args, **kwargs)
```

Comment:
starej super

Reply:
jo já vím, chamber měl ještě nedávno podporu 2.7 budu to muset odstranit všude..